packages/sveltekit/src/client/browserTracingIntegration.ts

```diff
@@ -76,8 +77,9 @@ function _instrumentPageload(client: Client): void {
       [SEMANTIC_ATTRIBUTE_SENTRY_SOURCE]: 'url',
     },
   });
-
-  const pageloadSpan = getActiveSpan();
+  if (!pageloadSpan) {
+    return;
+  }
```

Comment:
we can just early return here if we don't create a pageload span. No need to subscribe to the page store to update the name then.